src/pages/ScriptList/index.tsx

```diff
@@ -565,6 +699,20 @@ const ScriptTable: FC<{
       className: styles.colCreatedTime,
       render: (script: ScriptDetail) => parseSimpleDate(+script.deployedBlockTimestamp),
     },
+    {
+      key: 'action',
+      render: (script: ScriptDetail) => {
+        const codeHash = script.hashType === null ? script.typeHash : script.dataHash
```

Comment:
`dataHash` will be assigned to `codeHash` when `script.hashType === type`, which is unexpected.

Reply:
hashType only can be data data1 and data2

Reply:
`ScriptDetail.hashType` is defined as `string | null` at https://github.com/Magickbase/ckb-explorer-frontend/blob/develop/src/models/Script/index.ts#L11 and therotically it should be `type | data | data1 | data2`.
Does `null` equals to `type` in the response of API?